### DISCUSSION TITLE
can't install brew , RPC Failed

### DISCUSSION THREAD

**Discussion by yas1th** [Homebrew/discussions#1391] [Getting started]
I have used the **below command** to install home brew but facing the below error

**/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"**

* [new tag]             3.1.3                                               -> 3.1.3
 * [new tag]             3.1.4                                               -> 3.1.4
 * [new tag]             3.1.5                                               -> 3.1.5
HEAD is now at eae355810 Merge pull request #11297 from Homebrew/dependabot/bundler/docs/rexml-3.2.5
==> Tapping homebrew/core
remote: Enumerating objects: 950920, done.
remote: Counting objects: 100% (520/520), done.
remote: Compressing objects: 100% (242/242), done.
Receiving objects:  55% (524285/950920), 172.47 MiB | 5.93 MiB/s

**error: RPC failed; curl 56 LibreSSL SSL_read: SSL_ERROR_SYSCALL, errno 60**
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master

**Comment by rigmas** [comment]
I also facing this problem before. It solved by mirroring homebrew.
